ui/apps/platform/src/Containers/Policies/Wizard/Step5/PolicyEnforcementForm.tsx

```diff
@@ -54,12 +54,20 @@ function PolicyEnforcementForm() {
     const hasDeploy = values.lifecycleStages.includes('DEPLOY');
     const hasRuntime = values.lifecycleStages.includes('RUNTIME');
     const hasAuditLog = values.eventSource === 'AUDIT_LOG_EVENT';
+    const hasNodeEvent = values.eventSource === 'NODE_EVENT';
 
-    const responseMethodHelperText = showEnforcement
+    let responseMethodHelperText = showEnforcement
         ? 'Inform and enforce will execute enforcement behavior at the stages you select.'
-        : hasRuntime && hasAuditLog
-          ? 'Inform will always include violations for this policy in the violations list. Enforcement is not available for audit log event sources.'
-          : 'Inform will always include violations for this policy in the violations list.';
+        : 'Inform will always include violations for this policy in the violations list.';
+
+    if (hasAuditLog) {
+        responseMethodHelperText = 'Enforcement is not available for audit log event sources.';
+    }
+    if (hasNodeEvent) {
+        responseMethodHelperText = 'Enforcement is not available for node event sources.';
+    }
+
+    const isEnforcementDisabled = hasAuditLog || hasNodeEvent;
```

Comment:
`hasRuntime` is always true when either `hasAuditLog` or `hasNodeEvent` is true, so I removed the additional check here.